packages/runtime/container-runtime/src/summary/orderedClientElection.ts

```diff
@@ -604,7 +604,14 @@ export class OrderedClientElection
 			this._eligibleCount--;
 			if (this._electedClient === client) {
 				// Removing the _electedClient. There are 2 possible cases:
-				if (this._electedParent !== client) {
+				if (this._electedParent === client) {
+					// 2. The _electedClient is an interactive client that has left the quorum.
```

Comment:
Nit: fix "1." and "2." in the comments for each branch of the if.

Reply:
Good catch! The dangers of auto-fix...

Reply:
```suggestion
					// 1. The _electedClient is an interactive client that has left the quorum.
```